packages/host/app/lib/current-run.ts

```diff
@@ -389,6 +390,13 @@ export class CurrentRun {
         identityContext,
         realmPath: this.#realmPaths,
       });
+      embeddedHtml = await this.#renderCard({
```

Comment:
Hmm I thought in the boxel meeting we discussed rendering embedded cards with using the CardDef card type specifically so that all cards would look uniform in the cards grid. This would mean eschewing the card class that is part of the card and using card def specifically. @lukemelia am I correct in that thinking?

Reply:
Does this mean we also have to update the default template for the embedded format in `CardDef`? Currently, the default embedded template for `CardDef` is `MissingEmbeddedTemplate`, which basically informs the user that no embedded template is specified.

Reply:
Yeah, if that is the case the default template would match the cards grid item. We should probably talk about this at standup today.

Reply:
We should also talk about how to render the style for card chooser too.